app/models/administrative-unit.js

```diff
@@ -31,4 +51,103 @@ export default class AdministrativeUnitModel extends OrganizationModel {
     inverse: null,
   })
   scope;
+
+  get validationSchema() {
+    const REQUIRED_MESSAGE = 'Selecteer een optie';
+    return super.validationSchema.append({
+      classification: validateBelongsToRequired(REQUIRED_MESSAGE),
+      locatedWithin: validateBelongsToOptional(),
+      governingBodies: validateHasManyOptional(),
+      involvedBoards: validateHasManyOptional(),
+      exactMatch: validateBelongsToOptional(),
+      scope: validateBelongsToOptional(),
+      isAssociatedWith: validateRequiredWhenClassificationId(
+        [
+          ...WorshipServiceCodeList,
+          ...CentralWorshipServiceCodeList,
+          ...ApbCodeList,
+        ],
+        REQUIRED_MESSAGE
+      ),
+      hasParticipants: validateRequiredWhenClassificationId(
+        IGSCodeList,
+        REQUIRED_MESSAGE
+      ),
+      wasFoundedByOrganization: validateRequiredWhenClassificationId(
+        [...AgbCodeList, ...ApbCodeList],
+        REQUIRED_MESSAGE
+      ),
+      isSubOrganizationOf: validateRequiredWhenClassificationId(
+        [
+          ...AgbCodeList,
+          ...ApbCodeList,
+          ...IGSCodeList,
+          ...PoliceZoneCodeList,
+          ...AssistanceZoneCodeList,
+          ...WorshipServiceCodeList,
+          ...CentralWorshipServiceCodeList,
```

Comment:
In the current changeset-based validation we do not enforce this relation for worship units. I am not sure based on what you added it here and whether it might break something somewhere.